3D Chess/Assets/Scripts/PieceType.cs

```diff
@@ -0,0 +1,15 @@
+﻿public enum PieceType : int
```

Comment:
`int` is the default enum type, so there's no need to explicitly specify it here.

Reply:
Fixed with 6ca899c.